sdcm/nemesis.py

```diff
@@ -192,6 +193,11 @@ def run(self, interval=None):
         if interval:
             self.interval = interval * 60
         self.log.info('Interval: %s s', self.interval)
+
+        if self.__class__.__name__ == 'SisyphusMonkey':
```

Comment:
the logic is specific to  `SisyphusMonkey` so it should be in the  `SisyphusMonkey` class. Was there any specific reason that you decided not to put it in the Sisyphus monkey class?

Reply:
This is happening only in runtime, I couldn't put it in the SisypusMonkey class because I wanted it to happen only once.
Otherwise, it will happen every time nemesis starts.

Reply:
create a constructor in the SisyphusMonkey and run the methods as follows:
```python
class SisyphusMonkey(Nemesis):
    def __init__(self, *args, **kwargs):
        super(SisyphusMonkey, self).__init__(*args, **kwargs)
        self.get_all_disrupt_methods()
        self.shuffle_list_of_disruptions()
```